### PR TITLE
A fix for scrollbar out of border option

### LINKED ISSUE #5507: Improvement: Dropdown
The current Dropdown design is neat, except when there are multiple items and scrolling is needed. Which renders something like this:

image

The idea is to make the scroll bar within (inside the dropdown), so that it respects the width and border of the Dropdown.

### PR DESCRIPTION
**Pull request by BahugunaSaumya**
## Description

just fixed some scss styles for dropdown scrollbar made it thin  hopefully this is what was required

## Related Issues

https://github.com/nodejs/nodejs.org/issues/5507  fixes #5507 
### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.

- [x] I have run `npx turbo test` to check if all tests are passing, and/or `npx turbo test:snapshot` to update snapshots if I created and/or updated React Components.
- [x] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: components/Common/Dropdown/index.module.scss [hunk 1]
```diff
@@ -28,3 +27,16 @@
...
+.dropdownList::-webkit-scrollbar {
+  width: var(--space-06);
+}
+
+.dropdownList::-webkit-scrollbar-track {
+  background: transparent;
+}
+
+.dropdownList::-webkit-scrollbar-thumb {
+  background-color: rgba(155, 155, 155, 0.7);
```
Comment (ovflowd) [inline components/Common/Dropdown/index.module.scss]: Can we use a colour that we already have on our `tokens.scss`? And then apply an rgba out of that colour?

File: components/Common/Dropdown/index.module.scss [hunk 2]
```diff
@@ -28,3 +27,16 @@
     }
   }
 }
+.dropdownList::-webkit-scrollbar {
```
Comment (ovflowd) [inline components/Common/Dropdown/index.module.scss]: Can we add an empty space here?

File: components/Common/Dropdown/index.module.scss
Comment (ovflowd) [inline components/Common/Dropdown/index.module.scss:1]: These new styles should be inside `.dropdownList`... You can do something like

```scss
.dropdownList {
  ... already existing styles

  &::-webkit-scrollbar {
    ...
  }

  &::-webkit-scrollbar-track {
    ...
  }

  &::-webkit-scrollbar-thumb {
    ...
  }
}
```

**Review by mikeesto — APPROVED** [review]
This looks like an improvement to me. Thanks for your first contribution.

File: components/Common/Dropdown/index.module.scss [hunk 3]
```diff
@@ -14,17 +14,31 @@
...
       text-align: center;
       width: 100%;
-
-      &:hover {
-        background-color: var(--color-dropdown-hover);
-      }
     }
   }
+
+  &:hover {
```
Comment (ovflowd) [inline components/Common/Dropdown/index.module.scss]: This hover change may have been intentional. I guess you should revert it, as hover should only apply to the current entry you're hovering

File: components/Common/Dropdown/index.module.scss [hunk 4]
```diff
@@ -14,17 +14,30 @@
...
-      &:hover {
-        background-color: var(--color-dropdown-hover);
-      }
-    }
+  &:hover {
+    background-color: var(--color-dropdown-hover);
+   }
+  } 
+ } 
+  &::-webkit-scrollbar {
```
Comment (ovflowd) [inline components/Common/Dropdown/index.module.scss:30]:
```suggestion

  &::-webkit-scrollbar {
```

It's important to add an empty line after each statement

File: components/Common/Dropdown/index.module.scss [hunk 5]
```diff
@@ -14,17 +14,30 @@
...
+
+  &::-webkit-scrollbar-track {
+    background: transparent;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background-color: var(--color-text-secondary);
+    border: transparent;
+    border-radius: 20px;
+
```
Comment (ovflowd) [inline components/Common/Dropdown/index.module.scss]:
```suggestion
```

No need of an empty line here

File: styles/_tokens.scss [hunk 6]
```diff
@@ -177,4 +178,5 @@ body {
   --color-dropdown-background: var(--black10);
   --color-dropdown-hover: var(--black11);
   --color-blog-card-background: var(--black9);
+
```
Comment (ovflowd) [inline styles/_tokens.scss]:
```suggestion
```

File: styles/_tokens.scss [hunk 7]
```diff
@@ -82,6 +82,7 @@ body {
   --color-border-primary: var(--black4);
   --color-border-secondary: var(--black3);
   --color-border-accent: var(--color-brand-primary);
+
```
Comment (ovflowd) [inline styles/_tokens.scss]:
```suggestion
```

File: components/Common/Dropdown/index.module.scss
Comment (ovflowd) [inline components/Common/Dropdown/index.module.scss:1]: The indentation of the Sass in this file is wrong. I forgot prettier is not prettying these files, so let me update prettier config, then you can rebase this PR and run format.

**Review by ovflowd — APPROVED** [review]
LGTM! Thank you for this nice fix! Appreciate the contribution by a lot!

**Pull request merged**